pkg/apis/core/v1beta1/types.go

```diff
@@ -152,14 +152,33 @@ type DataVolumeSourceS3 struct {
 
 // DataVolumeSourceRegistry provides the parameters to create a Data Volume from an registry source
 type DataVolumeSourceRegistry struct {
-	//URL is the url of the Docker registry source
+	//URL is the url of the registry source (starting with docker:// for Docker, or is:// for ImageStream)
 	URL string `json:"url"`
+	//ImportMethod is the import method: "skopeo" (default), or "cri"
+	ImportMethod RegistryImportMethod `json:"importMethod,omitempty"`
 	//SecretRef provides the secret reference needed to access the Registry source
 	SecretRef string `json:"secretRef,omitempty"`
 	//CertConfigMap provides a reference to the Registry certs
 	CertConfigMap string `json:"certConfigMap,omitempty"`
 }
 
+const (
+	// RegistryTransportDocker is docker transport prefix
+	RegistryTransportDocker = "docker"
+	// RegistryTransportImageStream is image streamstransport prefix
+	RegistryTransportImageStream = "is"
+)
+
+// RegistryImportMethod represents the registry import method
+type RegistryImportMethod string
+
+const (
+	// RegistryImportSkopeo is the standard import method
+	RegistryImportSkopeo RegistryImportMethod = "skopeo"
```

Comment:
We don't actually use skopeo anymore, we use the containers library which has the same code that skopeo uses to do its thing, but no skopeo itself. The term used in the library is ImageSource, but I don't know if that is any clearer.

Reply:
Naming is always the hardest thing...
RegistryImportImageSource?
RegistryImportDocker?
RegistryImportCDI?

Reply:
Ack.  I like the naming suggested here.  PullMethod seems better since you could use either method against the same image source.  node and pod sound good to me.